src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=$MAIL_ACCOUNT_USERNAME
+spring.mail.password=$MAIL_ACCOUNT_PASSWORD
+
+spring.mail.properties.mail.smtp.auth=true
```

Comment:
You specified here all mail properties, why do you need your own JavaMailSender then?